lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -857,9 +846,19 @@ def _initialize_nodes(self, environment: Environment) -> None:
                 nic_map[name] = nic
 
         # get public IP
-        public_ip_address = network_client.public_ip_addresses.get(
-            environment_context.resource_group_name, RESOURCE_ID_PUBLIC_IP
-        ).ip_address
+        public_ip_addresses = network_client.public_ip_addresses.list(
```

Comment:
This is way easier to get with Azure CLI. Just load the JSON output of `az vm list-ip-addresses` and get the field.

Reply:
Yes, in this case, az looks easier to get public IPs.